src/components/HeaderBar/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import styles from "../HeaderBar.module.scss"
```

Comment:
Try to keep the `.scss` files for each component in the folder with its `index.tsx`. Once you move it in there, you can change this to `./HeaderBar.module.scss`